src/aap_eda/services/ruleset/activate_rulesets.py

```diff
@@ -74,6 +74,29 @@ def find_ports(rulebook_text: str):
     return found_ports
 
 
+def save_activation_and_instance(
```

Comment:
This function is kind of specific, should we define it as class method?

Reply:
I plan to use it from tasks.ruleset too.